persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -45,33 +45,7 @@ SELECT * FROM Member WHERE conversation = :conversation;
 
 selectConversationsByMember:
 SELECT
-    qualified_id,
-    name,
-    type,
-    team_id,
-    mls_group_id,
-    mls_group_state,
-    mls_epoch,
-    mls_proposal_timer,
-    protocol,
-    muted_status,
-    muted_time,
-    creator_id,
-    last_modified_date,
-    last_notified_date,
-    last_read_date,
-    access_list,
-    access_role_list,
-    mls_last_keying_material_update_date,
-    mls_cipher_suite,
-    receipt_mode,
-    message_timer,
-    user_message_timer,
-    archived,
-    archived_date_time,
-    verification_status,
-    proteus_verification_status,
-    legal_hold_status
+    Conversation.*
```

Comment:
just wondering, should we really use `*` in the queries? even though there is no change but to make more clear exactly was is being returned? 🤔

Reply:
The idea is to always return the same things as the Conversation table.

This way we can use the exact same mapper and it will always work.